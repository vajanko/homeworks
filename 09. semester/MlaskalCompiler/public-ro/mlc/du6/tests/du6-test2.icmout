ERROR: Jump out of code on IP=23
