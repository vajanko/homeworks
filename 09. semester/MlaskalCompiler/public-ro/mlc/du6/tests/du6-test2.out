Error (line 10): '1' is not a label
Error (line 13): Duplicated label '6' definition
Error (line 20): Duplicated label '6' definition
Error (line 3): Missing label '1' definition
Warning (line 3): Label '2' defined but never used
Warning (line 3): Label '4' declared but never used
Error (line 3): Missing label '5' definition
Warning (line 3): Label '6' defined but never used
Warning (line 7): Label '2' defined but never used
Warning (line 7): Label '4' declared but never used
Error (line 7): Missing label '5' definition
Warning (line 7): Label '6' defined but never used
There were some errors 6 and warnings 6
